e2e-tests/portal/cypress/e2e/journeys/checker/cover-end-date/deal.js

```diff
@@ -1,7 +1,8 @@
-const mockDeal = require('../../../../fixtures/deal');
-const CONSTANTS = require('../../../../fixtures/constants');
-const FIXTURE_CONSTANTS = require('../../../../../../e2e-fixtures/constants.fixture');
-const { today, tomorrow } = require('../../../../../../e2e-fixtures/dateConstants');
+import { BOND_FACILITY_TYPE } from '@ukef/dtfs2-common';
```

Comment:
If we are wanting to get this import change, we should get this in soon to avoid anything slipping through :)